python/lsst/ts/fbs/utils/auxtel/surveys.py

```diff
@@ -77,17 +103,30 @@ def generate_image_survey(
         additional_notes=[(target.target_name.split("_")[0], 32)],
     )
 
-    sequence = [ObservationArray(n=1) for i in range(len(target.filters))]
+    # Set up a sequence of nexp exposures per filter
+    # sequence_len = target.nexp * len(target.filters)
+    # sequence = ObservationArray(n=sequence_len)
+    # sequence["RA"] = target.ra.to(u.rad).value
+    # sequence["dec"] = target.dec.to(u.rad).value
+    # sequence["exptime"] = target.exptime
+    # sequence["nexp"] = 1
+    # sequence["scheduler_note"] = f"{target.survey_name}"
+    # sequence["target_name"] = f"{target.target_name}"
+    # sequence["science_program"] = f"{target.science_program}"
+    # i = 0
+    # for filter_obs in target.filters:
+    #     sequence["filter"][i : i + target.nexp] = filter_obs
+    #     i += target.nexp
 
+    # Use this method, until there is an update of rubin_scheduler at summit
```

Comment:
Not sure I understand this comment, is it to say that after an update at the summit we should use L107:L119 instead of this method? 

If so, I think we should include L107:119 in a separate branch as part of the update to rubin_scheduler. We could add a TO-DO here with a reference to  JIRA ticket if we want to leave some notes for the future.

Reply:
The newest rubin_scheduler handles the sequences/observation arrays a little differently (instead of a list of observation arrays, it allows a multi-element observation array). 
And also there is "band" instead of "filter" in the inputs. 
Do you want me to make another branch of this PR, that would be tested and used when rubin_scheduler gets updated at the summit? This was indeed sort of a to-do marker for me (and others)  in the future.

Reply:
Yeah a separate branch of this PR (which we can then rebase to develop when we merge) is probably the best way to go.